src/models/tiles/tile-model.test.ts

```diff
@@ -3,6 +3,17 @@ import { kDefaultMinWidth, TileModel } from "./tile-model";
 import { getTileTypes, getTileContentInfo } from "./tile-content-info";
 import { IUnknownContentModel } from "./unknown-content";
 import { kUnknownTileType } from "./unknown-types";
+import { ImageModel } from "./geometry/geometry-model";
+import placeholderImage from "../../../assets/image_placeholder.png";
+
+const editableTileTypes = [
+  "Table",
+  "Geometry",
+  "Image",
+  "Text",
+  "Drawing",
+  "Diagram"
+];
 
 // Define the built in tool ids explicitly as strings.
 // Strings are used because importing the tool id constant could trigger a
```

Comment:
Would it make sense to move `editableTileTypes` below this existing comment, and maybe modify the comment slightly? Seems like the comment is still sort of about the need to define those tool IDs as strings. Or am I misunderstanding something?

Reply:
Good idea.  Done in https://github.com/concord-consortium/collaborative-learning/pull/2628/commits/7ba7bd0fca27bd0868fb81e61880b5054fc70436.